website/releases/all-releases.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Releases
+sidebar_position: 1
+keywords: [hudi, download, release notes]
+toc: true
+last_modified_at: 2019-12-30T15:59:57-04:00
+---
+
+### Release 0.9.0
+* Release Notes : ([Release Notes for Apache Hudi 0.9.0](/releases/release-0.9.0))
+* Download Source : [Apache Hudi 0.9.0 Source Release](https://www.apache.org/dyn/closer.lua/hudi/0.9.0/hudi-0.9.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.9.0/hudi-0.9.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.9.0/hudi-0.9.0.src.tgz.sha512))
```

Comment:
@umehrot2 : I remember last time our announcement was rejected due to non usage of mirror system in download page. I see we have fixed all the source links, but still asc and sha512 is not using the mirror links right? should we fix that as well?

Reply:
nvm. cross checked [here](https://infra.apache.org/release-download-pages.html). we should be good. will wait for a day to see if Udit has any comments.

Reply:
Links wise we are fine. asc and sha512 are not supposed to be mirror links.